### PR TITLE
update to 7.20190730 and drop custom snapshot

### PR DESCRIPTION
**Pull request by notestaff**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ mkdir -p $STACK_ROOT
 ) > "${STACK_ROOT}/config.yaml"
 
 cp ${RECIPE_DIR}/git_annex_custom_snapshot.yaml .
```
Comment (yarikoptic) [inline recipe/build.sh]: <details>
<summary>fails here since file was removed (below in the diff)
</summary> 

```shell
+ cp /home/conda/recipe_root/git_annex_custom_snapshot.yaml .
cp: cannot stat '/home/conda/recipe_root/git_annex_custom_snapshot.yaml': No such file or directory
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2343, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1491, in build
    cwd=src_dir, stats=build_stats)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 397, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 377, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/git-annex_1564522988241/work/conda_build.sh']' returned non-zero exit status 1.
##[error]Bash exited with code '1'.
##[section]Finishing: Run docker build
```
</details>

**Comment by yarikoptic** [comment]
I guess as long as it builds, it is all good with me ;-) I hope to release datalad 0.11.6 later today/tomorrow which should also build/test finally ok, including against this bleeding edge annex morning precursor state (according to https://github.com/datalad/datalad/pull/3569)
